package com.swp.scijournal;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.scheduling.annotation.EnableScheduling;

@SpringBootApplication
@EnableScheduling 
public class ScientificJournalApplication {
    public static void main(String[] args) {
        SpringApplication.run(ScientificJournalApplication.class, args);
    }
}